apps/ledger-live-mobile/src/components/RootNavigator/WalletTabNavigator.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { useTheme } from "styled-components/native";
+import {
+  createMaterialTopTabNavigator,
+  MaterialTopTabBarProps,
+} from "@react-navigation/material-top-tabs";
+import styled from "@ledgerhq/native-ui/components/styled";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import { useSelector } from "react-redux";
+import { ScreenName } from "../../const";
+import Portfolio from "../../screens/Portfolio";
+import WalletNftGallery from "../../screens/Nft/WalletNftGallery";
+import { rgba } from "../../colors";
+import { readOnlyModeEnabledSelector } from "../../reducers/settings";
+import { accountsSelector } from "../../reducers/accounts";
+// eslint-disable-next-line import/no-cycle
+import ReadOnlyPortfolio from "../../screens/Portfolio/ReadOnly";
+
+const Tab = createMaterialTopTabNavigator();
+
+const TabBarContainer = styled(Flex)`
+  border-bottom-width: 1px;
+  border-bottom-color: ${p => p.theme.colors.palette.neutral.c40};
+  background-color: transparent;
+`;
+
+const StyledTab = styled.TouchableOpacity``;
+
+function TabBar({ state, descriptors, navigation }: MaterialTopTabBarProps) {
+  const { colors } = useTheme();
+
+  return (
+    <TabBarContainer
+      paddingLeft={4}
+      paddingRight={4}
+      paddingBottom={4}
+      paddingTop={4}
```

Comment:
p={4} no?

Reply:
YES

Reply:
ok, seems that this was on purpose because with the styled wrapper, just putting p={4} doesn't work ...